subsys/bluetooth/controller/ll_sw/nordic/lll/lll_scan.c

```diff
@@ -733,18 +731,19 @@ static inline u32_t isr_rx_pdu(struct lll_scan *lll, u8_t devmatch_ok,
 		/* assert if radio packet ptr is not set and radio started tx */
 		LL_ASSERT(!radio_is_ready());
 
-#if defined(CONFIG_BT_CTLR_PROFILE_ISR)
-		lll_prof_cputime_capture();
-#endif /* CONFIG_BT_CTLR_PROFILE_ISR */
+		if (IS_ENABLED(CONFIG_BT_CTLR_PROFILE_ISR)) {
+			lll_prof_cputime_capture();
+		}
 
-#if defined(CONFIG_BT_CTLR_GPIO_PA_PIN)
-#if defined(CONFIG_BT_CTLR_PROFILE_ISR)
-		/* PA/LNA enable is overwriting packet end used in ISR
-		 * profiling, hence back it up for later use.
-		 */
-		lll_prof_radio_end_backup();
-#endif /* CONFIG_BT_CTLR_PROFILE_ISR */
 
+#if defined(CONFIG_BT_CTLR_GPIO_PA_PIN)
```

Comment:
why not port this?

Reply:
Because of CONFIG_BT_CTLR_GPIO_PA_OFFSET. If BT_CTRL_GPIO_PA is not set then CONFIG_BT_CTLR_GPIO_PA_OFFSET is not defined